spec/controllers/proposals_controller_spec.rb

```diff
@@ -73,7 +73,7 @@
 
       before do
         expect(Proposal).to receive(:client_model_names).and_return(['SomeCompany::SomethingApprovable'])
-        expect(Proposal).to receive(:client_slugs).and_return(%w(some_company some_other_company))
+        expect(Proposal).to receive(:client_slugs).at_least(:once).and_return(%w(some_company some_other_company ncr))
```

Comment:
why `at_least` once? Also, feels off to me to have an expectation in a `before` block -- thoughts on moving into individual specs?